big_tests/rebar.lock

```diff
@@ -10,12 +10,12 @@
  {<<"ct_groups_summary_hook">>,
   {pkg,<<"ct_groups_summary_hook">>,<<"0.1.1">>},
   0},
- {<<"escalus">>,{pkg,<<"escalus">>,<<"4.3.0">>},0},
+ {<<"escalus">>,{pkg,<<"escalus">>,<<"4.5.0">>},0},
  {<<"esip">>,{pkg,<<"esip">>,<<"1.0.56">>},0},
  {<<"exml">>,{pkg,<<"hexml">>,<<"4.1.1">>},0},
- {<<"fast_pbkdf2">>,{pkg,<<"fast_pbkdf2">>,<<"1.0.6">>},2},
- {<<"fast_scram">>,{pkg,<<"fast_scram">>,<<"0.6.1">>},1},
- {<<"fast_tls">>,{pkg,<<"fast_tls">>,<<"1.1.21">>},1},
+ {<<"fast_pbkdf2">>,{pkg,<<"fast_pbkdf2">>,<<"2.0.0">>},2},
+ {<<"fast_scram">>,{pkg,<<"fast_scram">>,<<"0.7.0">>},1},
+ {<<"fast_tls">>,{pkg,<<"fast_tls">>,<<"1.1.22">>},1},
```

Comment:
Oh, if escalus 4.4 removed `fast_tls` entirely, how comes this is updated here? 😅